test/Libraries/Microsoft.Extensions.AI.Abstractions.Tests/Utilities/AIJsonUtilitiesTests.cs

```diff
@@ -234,7 +234,7 @@ public static void CreateParameterJsonSchema_TreatsIntegralTypesAsInteger_EvenWi
             """).RootElement;
 
         JsonSerializerOptions options = new(JsonSerializerOptions.Default) { NumberHandling = JsonNumberHandling.AllowReadingFromString };
-        AIFunction func = AIFunctionFactory.Create((int a, int? b, long c, short d) => { }, serializerOptions: options);
+        AIFunction func = AIFunctionFactory.Create((int a, int? b, long c, short d, float e, double f, decimal g) => { }, serializerOptions: options);
```

Comment:
What will happen if a type like Half or BigInteger is used?

Reply:
`Half` works the same as `float` and `double` for the targets that support it. `BigInteger` currently has no built-in support in STJ, its schema is an object containing all its public properties.